L1TriggerConfig/Utilities/src/L1TCaloParamsViewer.cc

```diff
@@ -218,13 +230,13 @@ void L1TCaloParamsViewer::analyze(const edm::Event& iEvent, const edm::EventSetu
 
     if( !ptr1->tauCalibrationLUT()->empty() ){
         cout<<"  tauCalibrationLUT=      ["<<ptr1->tauCalibrationLUT()->maxSize()<<"] "<<flush;
-        int tauCalibration[ptr1->tauCalibrationLUT()->maxSize()];
-        for(unsigned int i=0; i<ptr1->tauCalibrationLUT()->maxSize(); i++)
+        int tauCalibration[512]; //ptr1->tauCalibrationLUT()->maxSize()];
```

Comment:
hi @kkotov @rekovic  - while waiting for @ggovi..

this change is naively going in the wrong direction - are you trying to work around that the lack of a "size()" method in the LUT class and hardwiring some magic number that is currently "correct"?